### DISCUSSION TITLE
[2322010]God of War Ragnarök-战神诸神黄昏{本体}

### DISCUSSION THREAD

**Discussion by Niguyt5874** [pjy612/SteamManifestCache#839] [Share]
### Steam 游戏ID

2322010

### 密钥列表

```hash
2322011;78758c85a2a8b04770fb4fe430ac7a215a959b72e496d94f6a0ec5ac38066f49
2974340;b54ec4dbd1528067329c6ce102d3dc0501401e41b08a6cabc0c947d0c9541baf
```

### 清单附件

[2322010.zip](https://github.com/user-attachments/files/17078167/2322010.zip)

**Comment by 10086pro** [comment]
游戏三小时前更新了清单文件，您能重新上传一次清单文件吗？谢谢

**Comment by 10086pro** [comment]
更新后的清单文件
[Uploading 2322011_9175237445121531390.zip…]()

**Reply by Niguyt5874** [reply]
有没有可能一更新我就上传过了0.0

**Reply by 10086pro** [reply]
这游戏今天更新3次了，您上传的清单是-app 2322010 -depot 2322011 -manifest 8926344782481069122
在那之后又更新了两次，最新的清单是-app 2322010 -depot 2322011 -manifest 9175237445121531390
我给上传到您讨论区里了

**Reply by Niguyt5874** [reply]
不是啊,我10点24的时候更新了一次,我下自己的附件也是2322011_9175237445121531390的清单

**Reply by Niguyt5874** [reply]
文件创建时间10:24,我是连续更了两次

**Reply by Niguyt5874** [reply]
您讨论区的清单还没上传成功,显示正在上传

**Comment by 10086pro** [comment]
嗯嗯，那可能是文件更新有延迟。

**Reply by Niguyt5874** [reply]
那没事了(狗头)

**Comment by 10086pro** [comment]
谢谢大佬分享
